### PR TITLE
Create 2021-08-10-incubator-lesson-spotlight-julia-novice.md

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

**Comment by tobyhodges** [comment]
@BeastyBlacksmith @giordano @lucaferranti please review and approve/edit as you wish. The post is scheduled for publication on Tuesday 10 August [Edit: ideally we would get this merged by the end of Friday 6 August]. Thanks to @kmomar for putting together the pull request.

In particular, I wonder if you would like to expand a little on the _How You Can Help Develop This Lesson_ section? In my experience, people are more likely to respond to fairly specific prompts for contribution, e.g. you could list a few different types of contribution you would be happy to see to the lesson. Reviewing existing content, adding exercises, proposing new content, testing the material in a workshop and providing feedback, adding figures and other visual aids, proofreading, etc etc ...

File: _posts/2021/08/2021-08-10-incubator-lesson-spotlight-julia-novice.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* **Title:** {{page.lesson_title}}
+* **Lesson Pages:** https://carpentries-incubator.github.io/julia-novice/
+* **Lesson Repository:** https://github.com/carpentries-incubator/julia-novice
+
+### Learning Objectives
+
+* using the [Julia](https://julialang.org/) REPL
+* get comfortable with the package manager and environments
+* learn syntax for common tasks (types, control flow, functions & methods, modules)
+* finding, using and creating packages
```
Comment (BeastyBlacksmith) [inline _posts/2021/08/2021-08-10-incubator-lesson-spotlight-julia-novice.md]:
```suggestion
* finding, using and creating packages
* writing tests
```

File: _posts/2021/08/2021-08-10-incubator-lesson-spotlight-julia-novice.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+People with some programming experience that are new to Julia. Particularly people doing scientific computing.
+
+## Lesson Progress
+
+The lesson is in the pre-alpha stage at its first iteration and has yet to be taught.
+
+## How You Can Help Develop This Lesson
+
+We are looking for input in any form.
```
Comment (BeastyBlacksmith) [inline _posts/2021/08/2021-08-10-incubator-lesson-spotlight-julia-novice.md]:
```suggestion
We are looking for people
 * reviewing the lesson
 * teaching the lesson
 * adding content (e.g. multiple dispatch)
 * providing more visuals and interactive content
```

**Review by giordano — APPROVED** [review]
Looks good to me besides Simon's comments.  Thanks!

**Comment by lucaferranti** [comment]
LGTM, great work!

**Comment by tobyhodges** [comment]
Alright @kmomar this one is ready to go whenever it fits into the publication schedule.

**Pull request merged**